doc: matter: add QR codes to samples (#9857)
nrfconnect/sdk-nrf: doc/nrf/ug_matter_overview_commissioning.rst

* 27-bit setup passcode
* 8-bit Discovery Capabilities Bitmask

.. _ug_matter_network_topologies_commissioning_onboarding_formats:

Onboarding information formats
==============================

The data payload can be encoded in one of the following ways, either included on-device or in the packaging for the end products (or both):

* Manual Pairing Code - which provides the onboarding information as a sequence of digits that can be used with most Matter commissioners.
* QR Code - which you can scan using the appropriate mobile application for your ecosystem.

>   For testing purposes, this code can be printed to the UART console or be shared using an NFC tag.

Reviewer: This actually applies to QR Code Payload.

> * QR Code Payload - which is a code made of digits and letters that you can use in command-line tools.

Reviewer: Just throwing a shorter term, but it's up to you which version you prefer.
```suggestion
* QR Code Payload - which is an alphanumeric code that you can use in command-line tools.
```
